product_docs/docs/pgd/5/overview/index.mdx

```diff
@@ -59,7 +59,7 @@ Replication continues between currently connected nodes even if one or more node
 
 Nodes can run different release levels, negotiating the required protocols to communicate. As a result, EDB Postgres Distributed clusters can use rolling upgrades, even for [major versions](../upgrades/upgrading_major_rolling/) of database software.
 
-DDL is replicated across nodes by default. DDL execution can be user-controlled to allow rolling application upgrades if desired.
+DDL is replicated across nodes by default. DDL execution can be user controlled to allow rolling application upgrades, if desired.
```

Comment:
@jpe442 I noticed you edited to include the hyphen in one of your PRs. That's incorrect except when it's an adjective.